website/src/components/Edit/SequencesForm.spec.tsx

```diff
@@ -39,18 +14,16 @@ describe('SequencesForm', () => {
         vi.restoreAllMocks();
     });
     test('Empty editable sequences produces no output', () => {
-        const emptyEditableSequences = EditableSequences.fromSequenceNames(
-            makeReferenceGenomeLightweightSchema(['foo', 'bar']),
-        );
+        const MAX_SEQUENCES_PER_ENTRY = 1;
+        const emptyEditableSequences = EditableSequences.empty(MAX_SEQUENCES_PER_ENTRY);
 
         expect(emptyEditableSequences.getSequenceFasta()).toBeUndefined();
         expect(emptyEditableSequences.getSequenceRecord()).deep.equals({});
     });
 
     test('GIVEN organism with 2 suborganisms with 1 segment each THEN allows at max 1 inputs', async () => {
```

Comment:
we should change the test names as now the sequencesForm doesn't use the suborganisms